app/scripts/directives/mobileServiceInstanceList.js

```diff
@@ -36,7 +36,7 @@
     });
 
     ctrl.goToCatalog = function() {
-      Navigate.toProjectCatalog(ctrl.projectName, {category: 'mobile', subcategory: 'services'});
+      Navigate.toProjectCatalog($routeParams.project, {category: 'mobile', subcategory: 'services'});
```

Comment:
`ctrl.projectName` isn't defined. This results in the `Browse Catalog` button on directing to the main service catalog, rather than the catalog within the namespace.

Reply:
:+1: